changelog/changes/2023-10-25-docker-gentoo-upstream.md

```diff
@@ -1,4 +1,4 @@
-- cri-tools, runc, containerd, docker, and docker-cli are now shipped without debugging symbols and built from Gentoo upstream ebuilds. Docker was updated to Docker 24 (see "updates").
+- cri-tools, runc, containerd, docker, and docker-cli are now built from Gentoo upstream ebuilds. Docker received a major version upgrade - it was updated to Docker 24 (from Docker 20; see “updates”).
```

Comment:
I think this makes this line show up in the next Alpha but we can fix this by hand

Reply:
The show-changes script works by emitting all files under `changelog/` in the new release that have been created/touched compared to the old release.